tests/Schema/Keywords/TypeTest.php

```diff
@@ -62,11 +62,8 @@ public function testItValidatesTypeRed() : void
 
             $schema = $this->loadRawSchema($spec);
 
-            try {
-                (new Validator($schema, $invalidValue))->validate();
-            } catch (ValidationKeywordFailed $e) {
-                $this->assertEquals('type', $e->keyword());
-            }
+            $this->expectException(TypeMismatch::class);
```

Comment:
Some tests becase a bit more clear, where exception inspection is not needed

Reply:
Yeah, looks good